src/radical/pilot/agent/lm/dplace.py

```diff
@@ -42,9 +42,9 @@ def construct_command(self, cu, launch_script_hop):
             raise RuntimeError('insufficient information to launch via %s: %s' \
                     % (self.name, slots))
 
-        task_offsets = slots['task_offsets']
-        assert(len(task_offsets) == 1)
-        dplace_offset = task_offsets[0]
+        task_offsets = slots['task_offsets']        # This needs to revisited 
+        assert(len(task_offsets) == 1)              # since slots structure has 
+        dplace_offset = task_offsets[0]             # changed
```

Comment:
@iparask  also here

Reply:
I believe this is not part of this PR and is part of devel. @andre-merzky can you verify?

Reply:
agree - that was already discussed with @georgeha .